en-US/firefox/welcome/page4.lang

```diff
@@ -0,0 +1,69 @@
+## NOTE: demo page available at https://www-dev.allizom.org/firefox/welcome/4/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/welcome/4/
+
+# HTML page title
+;Download the Firefox Browser on your Mobile for iOS and Android
+Download the Firefox Browser on your Mobile for iOS and Android
+
+
+;Wallet. Keys. Phone. Firefox.
+Wallet. Keys. Phone. Firefox.
+
+
+;Take privacy with you on every device — and leave the data trackers behind.
+Take privacy with you on every device — and leave the data trackers behind.
+
+
+;Get the Firefox App
+Get the Firefox App
+
+
+;Get Firefox on your Phone
+Get Firefox on your Phone
+
+
+;Send the download link right to your phone or email.
+Send the download link right to your phone or email.
+
+
+;Enter your phone number or email
+Enter your phone number or email
+
+
+# This string is reused from firefox/whatsnew
```

Comment:
@peiying2 this string will not be exposed to the `send_to_device` locales:
 - `'de'` 
- `'en'`
- `'es-AR', 'es-CL', 'es-ES', 'es-MX'` 
- `'fr'`
- `'id'`
- `'pl'`
- `'pt-BR'`
- `'ru'` 
- `'zh-TW'`

Reply:
Is this page opened to all the mozorg locales?

Reply:
@peiying2 it will be opt-in. Just wanted to mention these locales wouldn't see this string.